test.js

```diff
@@ -56,11 +56,30 @@ test('should throw for empty "unreleased" section', async t => {
   await assert.rejects(runTasks(plugin), /There are no entries under "Unreleased" section in CHANGELOG-EMPTY\.md/);
 });
 
-test('should not throw for empty "unreleased" section when no-increment flag is set', async t => {
-  const options = { increment: false, [namespace]: { filename: 'CHANGELOG-EMPTY.md' } };
+test('should throw for missing "unreleased" section when no-increment flag is set if changelog is misformatted', async t => {
+  const options = { isIncrement: false, [namespace]: { filename: 'CHANGELOG-FOO.md' } };
```

Comment:
`isIncrement: false` on these options didn't seem to actually change the value on the config when I debugged the test. I'm not sure how to set this value correctly and i couldn't find any example tests using isDryRun (the only other variable coming off the base config)

Reply:
The option to set is `increment: false`, the way to read it from the main config is `this.config.isIncrement`

Reply:
Updated! That fixed up all the tests, thanks!